### PR TITLE
Sidebar edit

### PR DESCRIPTION
**Pull request by jacildoaldrin**

### CODE REVIEW

File: src/containers/Layout/Layout.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ const Layout = (props) => {
     setToggled(false);
   };
 
+  document.body.style.overflow = toggled ? "hidden" : "unset";
```
Comment (AshenCat) [inline src/containers/Layout/Layout.js:26]: hmmm... is the purpose of this to prevent scrolling?

**Review by AshenCat — APPROVED** [review]

**Review by lilianyangc — APPROVED** [review]

File: src/containers/Layout/Layout.js (hunk 1 above)
Comment (jacildoaldrin) [reply src/containers/Layout/Layout.js:26]: yep

Comment (victafrancis) [reply src/containers/Layout/Layout.js:26]: what's happening here now? lol what's changing

Comment (victafrancis) [reply src/containers/Layout/Layout.js:26]: oh hi guys, pwede ko pala kayo chat from here lol

Comment (lilianyangc) [reply src/containers/Layout/Layout.js:26]: Lolz

**Review by victafrancis — APPROVED** [review]

**Pull request merged**